carbon-design-system/carbon
# docs(project): add nextjs example

This PR adds in a nextjs example for Carbon in the examples folder

#### Changelog

**New**

- Add nextjs example 

**Changed**

**Removed**

#### Testing / Reviewing

- Checkout PR
- Run `yarn`
- Run `cd examples/nextjs`
- Run `yarn dev`
- Visit `http://localhost:3000` and verify that the page compiles and includes a Carbon button
